packages/core/src/contexts/transactionsHistory/models/transaction/parser/parsers/renbtc/const.ts

```diff
@@ -0,0 +1,4 @@
+export const REN_BTC_MINT = [
```

Comment:
I think this file should be named like `constants.ts` like in other folders of projects